src/components/markdown/Document.js

```diff
@@ -4,11 +4,13 @@ import Editor from './Editor';
 import store from '../../store';
 import { getMarkdown } from '../../selectors/selectMarkdown'; 
 import { updateMarkdown } from '../../actions/actionUpdateMarkdown';
+import MarkdownFile from './MarkdownFile';
 import styles from './Document.css';
 
 export default class Document extends PureComponent {
   state = {
     markdown: '# Hi there',
+    markdownTitles: ['tab1', 'tab2'],
```

Comment:
These titles should be coming from the store. To complete this ticket, you'll need to complete the ticket "Create container to save markdown files (4pts)". Can't really do one without the other.